src/app/Http/Controllers/Backend/Setup/FeeAmountController.php

```
@@ -0,0 +1,181 @@
+        }
+    }
+
+    public function EditFeeAmount($id){
+        $doc = new stdClass();
+        $doc = FeeCategoryAmount::where('fee_category_id', $id)->orderBy('class_id', 'asc')->get();
+        $doc->fee_categories = FeeCategory::all();
+        $doc->classes = StudentClass::all();
+        return view('backend.setup.fee_amount.edit-fee',  ['doc' => $doc]);
+    }
+
+    public function UpdateFeeAmount(UpdateFeeAmountRequest $request, $id)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$id'. (Unused Code Rules)

(UnusedFormalParameter)